### PR TITLE
closes #3 -  removes harry potter references

### LINKED ISSUE #3: inclusive language
harry potter references

### PR DESCRIPTION
**Pull request by graciemcguire**
changes harry potter / hogwarts references to reference art disciplines and famous qtpoc artists instead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,54 +156,56 @@ The output of the `reduce()` method does not need to be a primitive value like a
...
+  "Theater": [],
+  "Writing": []
 }
 
 ```
 
-Imagine we also have a `sortingHat` object that includes a `houseAssign()` method.
-That method takes the name of a student as its argument and returns the name of
-the house the student should be assigned to. We could call this method directly:
+Imagine we also have a `sortingObject` object that includes a `showcaseAssign()`
```
Comment (lizbur10) [inline README.md]: It's a little harder to follow without the background understanding of what the sorting hat does. Maybe call the object `studentSorter` instead to make it a little more explicit.

File: README.md [hunk 2]
```diff
@@ -156,54 +156,56 @@ The output of the `reduce()` method does not need to be a primitive value like a
...
-the current student name as its argument. `houseAssign()` returns the name of a
-Hogwarts house, which is then used as the key to access the correct array in the
-`hogwartsHouses` object and push the student's name into it. The iteration then
-continues to the next element in the array, passing the next student name and
-the updated value of `hogwartsHouses` as the arguments. Once `reduce()` has
-iterated through all the students in `incomingStudents`, it returns the final
-value of `hogwartsHouses`.
+in turn. Inside the callback, the `sortingObject.showcaseAssign()` method is
+called with the current student name as its argument. `showcaseAssign()` returns
+the name of a Arts Showcase, which is then used as the key to access the
```
Comment (lizbur10) [inline README.md]:
```suggestion
the name of an Arts Showcase, which is then used as the key to access the
```

**Review by lizbur10 — COMMENTED** [review]
It looks good! I made one small suggestion and fixed a typo

File: README.md (hunk 1 above)
Comment (graciemcguire) [reply README.md]: good call!

**Pull request merged**